packages/repo-tools/package.json

```diff
@@ -51,6 +52,16 @@
     "@types/mock-fs": "^4.13.0",
     "mock-fs": "^5.1.0"
   },
+  "peerDependencies": {
+    "@rushstack/node-core-library": "*",
```

Comment:
not sure about it, but we need the exact same version as in the transitive dependecies from `@microsoft/api-extractor` and `@microsoft/api-documenter` if not we get the same issue with `material-ui` packages, since we are monkey  patching different versions

Reply:
the alternative is to fix the version, thinking about what is better tbh